src/components/App/App.js

```diff
@@ -23,10 +29,22 @@ export default function App() {
       characters = data
     })
     .catch(error => handleGetCharacterError(error))
+
+  function onSearch(searchString) {
+    const filteredCharacters = characters.filter(character =>
+      character.name.includes(searchString)
+    )
+    createCards(filteredCharacters)
+  }
   function onFilterByHouse(house) {
-    console.log('App says: ', house)
     const filteredCharacters = characters.filter(
-      character => character.house === house
+      character => house == null || character.house === house
+    )
+    createCards(filteredCharacters)
+  }
+  function onFilterByGender(gender) {
```

Comment:
The `onFilterByHouse` and rhe `onFilterByGender` functions are identical, just the parameter changes. Maybe you could try and write one function `onFilter`?